### ISSUE TITLE
[Add-on] In high contrast mode, the door-hanger is not properly displayed

### ISSUE THREAD

**Issue by sbadau** [mozilla/fx-private-relay#188]
**Affected versions**
- Nightly 77.0a1
- Private Relay add-on version 1.2.0 - https://drive.google.com/file/d/1OqZCFEwHrbUTH0q1dQUJBI43qVBaM3Hw/view
                                                                                                                                                                                                                                             
**Affected platforms**
- Windows 10


**Steps to reproduce**
1. Enable the High Contrast mode 
2. Launch the latest Nightly 77.0a1
3. Install the Private Relay add-on and click on it
4. Observe the Firefox Private Relay door-hanger

**Expected behavior**
The door-hanger should be properly displayed.

**Actual behavior**
The door-hanger is not properly displayed in High Contrast Mode.

Please see the screenshot for more details:
![2020-04-23_18h21_43](https://user-images.githubusercontent.com/56026823/80117774-41909280-8590-11ea-89b8-a71aa2e7c2d2.png)

**Comment by sbadau** [comment]
The issue is still reproducible on the latest version of the add-on. Please see the screenshot.
![2020-06-19_18h39_40](https://user-images.githubusercontent.com/56026823/85151338-55890580-b25c-11ea-9126-a13e0b5535c3.png)
